src/Fixer/FunctionNotation/NoSpacesAfterFunctionNameFixer.php

```diff
@@ -145,40 +133,21 @@ private function getFunctionyTokenKinds()
     {
         static $tokens = [
             T_ARRAY,
-            T_ECHO,
+            //T_ECHO,
```

Comment:
Please remove the lines entirely instead of using comments.

Reply:
Thanks for reviewing, done.